Cargo.toml

```diff
@@ -50,10 +50,10 @@ async-recursion = "1.0.0"
 async-trait = "0.1.53"
 bincode = "1.3.3"
 bytes = "1"
-chrono = { version = "0.4.23", default-features = false, features = ["clock"] }
+chrono = { version = "0.4.22", default-features = false, features = ["clock"] }
 codemap = "0.1.3"
 codemap-diagnostic = "0.1.1"
-deno_core = "0.174.0"
+deno_core = { git = "https://github.com/exograph/deno.git", branch = "patched" }
```

Comment:
We should pin the dependency, perhaps by specifying "rev" (https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories)

Reply:
It's auto-pinned in the lockfile!